lib/Config.php

```diff
@@ -44,7 +44,7 @@ public function __construct(IConfig $config, IRequest $request) {
 
 	/**
 	 * @param array $parameters
-	 * @return \OC_OCS_Result
```

Comment:
``OC_OCS_Result`` is deprecated. ``phan`` reported this in lots of places. Easily fixed.